### DISCUSSION TITLE
How to create this script

### DISCUSSION THREAD

**Discussion by niestadt** [OlivierZal/com.melcloud#152] [Q&A] [answered]
Hello Olivier,
Somehow I can’t see how to write this use case:
If my son sets the target temperature of the airco on his room below a certain degree in summer, or above a certain degree in winter, then the target temperature is automatically adjusted to the lowest or highest allowed value. 

The airco is in automatic mode now. 

I did create such a flow in the past, but somehow it has disappeared and I don’t know how to create it again. 

Thanks in advance for your reply 😃

Regs, Dirk

**Comment by OlivierZal** [comment]
Hi @niestadt 

Here is a suggestion how to proceed with an advanced workflow:

WHEN the measure temperature has changed
IF measure temperature < lowest_temperature THEN target temperature = lowest_defined_temperature
IF measure temperature > highest_temperature THEN target temperature = highest_defined_temperature

Does this address your need?

**Reply by niestadt** [reply, marked as answer]
Hi, thanks for your advice. This was not possible in the Dutch version that I make use of. But it inspired me for this solution that help. I hope it does work. 

WHEN temperature gets < x degree
THEN set target temperature = x degree

The “if measure temperature” is not available in my version. 

But I believe that it will work now.
Thanks!

**Reply by OlivierZal** [reply]
You’re right! That’s a good way to do, I forgot that there are temperature `when` cards including a `<` / `>` condition. 

FYI I’ve just check: `if` card unavailability is not related to Dutch but to the fact that the app uses the Homey standard flows for temperatures capabilities, and since a `if` is included in `when` cards, the specific `if` card does not exist. 

However it is possible to create such `if` cards by using the `logic` category (and not the `MELCloud` one).
